File: 4th Semester/Systems for Design and Implementation/Java/Homework3/src/main/java/OnwardTravel/models/ClientSeat.java
package OnwardTravel.models;

public class ClientSeat {
    private final Client client;
    private final Reservation reservation;
    private final ReservedSeat seat;

    public ClientSeat(Client client, Reservation reservation, ReservedSeat seat) {
        this.client = client;
        this.reservation = reservation;
        this.seat = seat;
    }

    public Client getClient() {
        return client;
    }

    public Reservation getReservation() {
        return reservation;
    }

    public ReservedSeat getSeat() {
        return seat;
    }
}

File: 4th Semester/Systems for Design and Implementation/C#/Homework4/Models/ClientSeat.cs
using System;

namespace OnwardTravel.Models
{
    [Serializable]
    public class ClientSeat
    {
        public Client Client { get; }
        public Reservation Reservation { get; }
        public ReservedSeat ReservedSeat { get; }

        public ClientSeat(Client client, Reservation reservation, ReservedSeat reservedSeat)
        {
            Client = client;
            Reservation = reservation;
            ReservedSeat = reservedSeat;
        }
    }
}
